src/Core/Settings.cpp

```diff
@@ -5871,6 +5871,9 @@ Allows creation of [JSON](../../sql-reference/data-types/newjson.md) data type.
 )", BETA) ALIAS(enable_json_type) \
     DECLARE(Bool, allow_general_join_planning, true, R"(
 Allows a more general join planning algorithm that can handle more complex conditions, but only works with hash join. If hash join is not enabled, then the usual join planning algorithm is used regardless of the value of this setting.
+)", 0) \
+    DECLARE(Bool, validate_enum_literals_in_opearators, false, R"(
```

Comment:
`opearators` --> `operators`

Reply:
Thanks for catching it before the release, fix https://github.com/ClickHouse/ClickHouse/pull/74505